README.md

```diff
@@ -1,2 +1,92 @@
 # AR-Map-Based-Projector
 Srishti 2019
+
+## Image of the project
+
+<img src="Images%20and%20Videos/Images/ARSandbox.png" width="400" >
+
+## ABSTRACT:
+
+A R Based Map Projector A R Sandbox is hands-on sandbox combined with 3 D visualisation applications where sand is augmented in real-time with an elevation color map, topographic contour lines, and simulated water.Simply mold the sand by hand and the landscape comes to life! The AR Sandbox uses a 3D camera such as the Microsoft Kinect to scan the 3D shape of the sand surface in real time.A computer collects data from the camera, and creates a real-time dynamic topography map using the Processing software program. A projector, also mounted above the sandbox, projects that topography map back onto the real sand, calibrated such that real and virtual features line up exactly.
+
+## Motivation
+Aerial Mapping and surveying.
+
+## Mechanical Aspect of the Design:
+1. Sandbox of dimensions(75*60cm)
+2. Stand for kinect (mount on the plywood base of sandbox ,height 80-100cm,30cm long card board with a slot to mount kinect)
+3. Stand for projector opposite to kinect stand(height= 170cm)
+4. Table for laptop
+5. White sand
+
+## Electronics Aspect of the Design:
+1. Extension board 
+2. Microsoft kinect v2
+3. Benq projector
+4. Hdmi cable
+
+## Softwares used
+1. kinect SDK
+2. KinectConfigurationVerifier
+3. processing-3.5.3
+4. Kinect Pv2 library
+
+
+## Cost Structure:
+
+|components  |    cost       |
+|------------|-------------| 
+| Kinect           |5000 Rs   | 
+| Projector           |22000 Rs| 
+| white sand           |500 Rs|
+|nut,bolt,aluminium rod,L joints|300 Rs|
+
+
+
+
+## Applications:
+
+- Dynamic projection
+- Projection on uneven surface
+- Projection on building
+- Military application
+- In oil industries
+- To determine water flow pattern in valleys(dam construction)
+
+## Limitations:
+
+Range of depth sensor is accurate in range of 50cm-4.5m
+
+## Future Improvements:
+
+1. Perspective transformation to make depth mapping more accurate.
+2. add water flow animation to study water flow in valleys
+
+## Team members:
+
+1. Shubham Malviya  
+2. Shubhang Pandey
+
+<img src="https://user-images.githubusercontent.com/48958222/55109353-abbfd500-50fb-11e9-9ba8-3227992f6d11.jpeg" width="250">
+
+
+## Mentors:
+1. Kunal Kumar
+2. Shreeyash Geda
+
+## References
+For more information, visit[ AR Sandbox](https://arsandbox.ucdavis.edu/)
```

Comment:
remove this link instead  of this post the video on youtube and paste that link of your project.